pom.xml

```diff
@@ -95,6 +95,12 @@
                 <module>vaadin-platform-hybrid-test</module>
             </modules>
         </profile>
+        <profile>
+            <id>hilla-hybrid</id>
```

Comment:
"hybrod-security" would be a better name probably.

Reply:
done